react-game-project/src/App.css

```diff
@@ -0,0 +1,27 @@
+.App {
+  text-align: center;
+}
+
+.headerText {
+  color: black
+}
+
+.guessButton {
+  font-family: Verdana, Geneva, Tahoma, sans-serif;
+  border: 1px solid black;
+  margin-left: 5px;
+  background-color:black;
+  color: white;
+}
+
+.guessButton:hover {
+  font-family: Verdana, Geneva, Tahoma, sans-serif;
+  border: 1px solid black;
+  margin-left: 5px;
+  background-color: red;
+  color: black;
+}
+
```

Comment:
Is red a tacky color for the guessesLeftDiv ? @ndp

Reply:
Red is very nice, there are websites for color matching :)